galaxy/Dockerfile

```diff
@@ -89,6 +89,7 @@ ADD roles/ /tmp/ansible/roles
 ADD provision.yml /tmp/ansible/provision.yml
 ADD postgresql_provision.yml /tmp/ansible/postgresql_provision.yml
 RUN ansible-playbook /tmp/ansible/postgresql_provision.yml && \
+    echo "include_dir 'conf.d'" >> /var/lib/postgresql/9.3/main/postgresql.conf && \
```

Comment:
Indeed, that should do it, but why don't you export the postgresql.conf on startup?
The file /etc/postgresql/9.3/main/postgresql.conf already has that line ...